rcl/QUALITY_DECLARATION.md

```diff
@@ -101,7 +101,7 @@ Currently nightly test results can be seen here:
 
 ### Public API Testing [4.ii]
 
-Most of the functionality of the declared API in this package is covered in its unit tests. Currently it has a line coverage of [77%](https://ci.ros2.org/job/ci_linux_coverage/85/cobertura/src_ros2_rcl_rcl_src_rcl/).
+Most of the functionality of the declared API in this package is covered in its unit tests. Currently it has a line coverage of [80%](https://ci.ros2.org/job/ci_linux_coverage/lastBuild/cobertura/src_ros2_rcl_rcl_src_rcl/).
```

Comment:
Actually the `loaned_messages` are not being tested under CI (because it is not supported by Tier1 RMW providers). Mind adding a note to the API docs about this? We discussed this over Slack @chapulina 

Something like "The following functions are partially supported: `rcl_take_loaned_message`, `rcl_return_loaned_message_from_subscription`, `rcl_borrow_loaned_message`, `rcl_return_loaned_message_from_publisher` and `rcl_publish_loaned_message` because they are not currently supported on Tier 1 RMW providers."